tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -918,7 +918,7 @@
 
 CONTROLLER_OUTPUT_SIGNAL_PROPERTIES_DESCRIPTION = (
     case_description.ControllerOutputSignalPropertiesDescription(
-        controlled_property="opening",
+        controlled_property="valve current opening",
```

Comment:
IIUC, the proper grammar is:


```suggestion
        controlled_property="currently opening valve",
```